src/pybind/mgr/orchestrator.py

```diff
@@ -183,19 +183,21 @@ def wait(self, completions):
         """
         raise NotImplementedError()
 
-    def add_host(self, host):
-        # type: (str) -> WriteCompletion
+    def add_host(self, host, role=None):
+        # type: (str, str) -> WriteCompletion
         """
         Add a host to the orchestrator inventory.
         :param host: hostname
+        :param role: Role to performn (osd, mon, mgr)
         """
         raise NotImplementedError()
 
-    def remote_host(self, host):
-        # type: (str) -> WriteCompletion
+    def remove_host(self, host, role=None):
```

Comment:
why do we need the role when removing hosts? For a user this looks like a redundant information that I need to pass through.

Reply:
Removed